src/main/java/me/spartacus04/jext/listeners/JukeboxClickEvent.kt

```diff
@@ -67,11 +59,9 @@ internal class JukeboxClickEvent : JextListener() {
     private fun jukeboxGui(event: PlayerInteractEvent, block: Block) {
         event.isCancelled = true
         if(INTEGRATIONS.hasJukeboxGuiAccess(event.player, block)==false) {
-            event.player.sendMessage("Missing flag")
             return
         }
         if(!(event.player.hasPermission("jext.usejukebox")||event.player.isOp)){
-            event.player.sendMessage("Missing permission")
             return
         }
```

Comment:
Just like before, this check should be moved to the OnJukeboxInteract method, you can also remove the || event.player.isOp since it should be managed by a permission manager and not the plugin